pool/sql_queries.go

```diff
@@ -73,6 +73,15 @@ const (
 		confirmed BOOLEAN NOT NULL
 	);`
 
+	purgeDB = `DROP TABLE IF EXISTS 
```

Comment:
This query already exists in `pool_test.go` and shouldn't be duplicated. We probably need to remove the other one and keep this one.